src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie;
+  movie: Movie | null;
```

Comment:
you have already checked that movie is not null  in FindMovie component
```suggestion
  movie: Movie;
```